desktop/app/lib/session/index.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * External Dependencies
+ */
+const EventEmitter = require( 'events' ).EventEmitter;
+
+/**
+ * Internal Dependencies
+ */
+const keychain = require( '../../lib/keychain' );
+const log = require( '../../lib/logger' )( 'desktop:session' );
+
+/*
+ * Module constants
+ */
+
+class SessionManager extends EventEmitter {
+	constructor() {
+		super();
+	}
+
+	async init( window ) {
+		log.info( 'Initializing session manager...' );
+
+		this.loggedIn = false;
+		this.window = window;
+
+		// Check for existing cookies
+		const wordpress_logged_in = await getCookie(
```

Comment:
Why not camel case?

Reply:
The intent was to exactly reflect the names of the cookies being queried, as I thought that made it a little more intuitive.